ci/tests/puppeteer/cas.js

```diff
@@ -327,6 +364,22 @@ exports.launchSamlSp = async (idpMetadataPath, samlSpDir, samlOpts = []) => {
     return exec;
 }
 
+exports.stopActuator = async (baseUrl) => {
+    let args = ['-k', '-X', 'POST', `${baseUrl}/actuator/shutdown`];
+    await this.logg(`Stopping boot app via shutdown actuator with ${args}`);
+    const exec = spawn("curl", args);
```

Comment:
I dont think this is in fact necessary. You can send a POST using other functions that are here. No need to spawn curl.